pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -415,6 +415,24 @@ func (l *Launcher) GetDomain(_ context.Context, _ *cmdv1.EmptyRequest) (*cmdv1.D
 
 	return response, nil
 }
+func (l *Launcher) GetQemuVersion(_ context.Context, _ *cmdv1.EmptyRequest) (*cmdv1.QemuVersionResponse, error) {
+	response := &cmdv1.QemuVersionResponse{
+		Response: &cmdv1.Response{
+			Success: true,
```

Comment:
Be defensive in this function, use default value of `false` for Success (do not specify it) and instead set it to true if no error occured. You can reduce the if/else to something like:

```
if err != nil {
    response.Response.Message = getErrorMessage(err)
} else {
    response.Response.Success = true
    response.Version = version
}
```

Reply:
Nice one! Done.